joint1_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: bahu_putar_kanan
    joint_speed: 1.17
    motor:
        id: 1
        init: 2048
        min: 0
        max: 4095

joint2_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: bahu_putar_kiri
    joint_speed: 1.17
    motor:
        id: 2
        init: 2048
        min: 0
        max: 4095

joint3_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: bahu_angkat_kanan
    joint_speed: 1.17
    motor:
        id: 3
        init: 2048
        min: 0
        max: 4095

joint4_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: bahu_angkat_kiri
    joint_speed: 1.17
    motor:
        id: 4
        init: 2048
        min: 0
        max: 4095

joint5_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: siku_kanan
    joint_speed: 1.17
    motor:
        id: 5
        init: 2048
        min: 0
        max: 4095

joint6_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: siku_kiri
    joint_speed: 1.17
    motor:
        id: 6
        init: 2048
        min: 0
        max: 4095
